### PR TITLE
up ctx: Add menu item for connecting to a control plane

### PR DESCRIPTION
**Pull request by cbuto**
### Description of your changes

This PR adds a menu item to the space/group/control plane views that allows users to select it which saves the context and exits, just like q/f10.

Fixes https://github.com/upbound/up/issues/502

I have:

- [x] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR, as appropriate.

### How has this code been tested

tested locally

screenshots:

Control plane:
Screenshot 2024-05-10 at 4 54 48 PM

Group:
Screenshot 2024-05-10 at 4 53 42 PM

Space:
Screenshot 2024-05-10 at 4 53 06 PM

### CODE REVIEW

File: cmd/up/ctx/navigation.go [hunk 1]
```diff
@@ -466,15 +466,13 @@ var _ Back = &ControlPlane{}
...
-		/*
-			item{text: fmt.Sprintf("Connect to %s", ctp.NamespacedName), onEnter: KeyFunc(func(ctx context.Context, upCtx *upbound.Context, m model) (model, error) {
-				msg, err := ctp.Accept(ctx, upCtx)
-				if err != nil {
-					return m, err
-				}
-				return m.WithTermination(msg, nil), nil
-			}), padding: []int{1, 0, 0}},
-		*/
+		item{text: fmt.Sprintf("Connect to %s", ctp.NamespacedName().Name), onEnter: KeyFunc(func(m model) (model, error) {
```
Comment (cbuto) [inline cmd/up/ctx/navigation.go:469]: do we want something like this on every level? cc @RedbackThomson @cwilhit

Comment (RedbackThomson) [inline cmd/up/ctx/navigation.go:469]: "Connect to x and quit"?

Comment (cwilhit) [reply cmd/up/ctx/navigation.go:469]: yep

File: cmd/up/ctx/navigation.go [hunk 2]
```diff
@@ -245,9 +245,17 @@ func (s *Space) Items(ctx context.Context, upCtx *upbound.Context) ([]list.Item,
 	}
 
 	if len(nss.Items) == 0 {
-		items = append(items, item{text: "No groups found", emptyList: true})
+		items = append(items, item{text: "No groups found", notSelectable: true})
 	}
 
+	items = append(items, item{text: fmt.Sprintf("Switch kubeconfig to %q and quit", s.Name), onEnter: func(m model) (model, error) {
```
Comment (cbuto) [inline cmd/up/ctx/navigation.go:251]: not attached to this wording at all, @cwilhit @RedbackThomson lemme know if you have any ideas

File: cmd/up/ctx/navigation.go [hunk 3]
```diff
@@ -245,9 +245,17 @@ func (s *Space) Items(ctx context.Context, upCtx *upbound.Context) ([]list.Item,
...
+		msg, err := s.Accept(m.upCtx, m.contextWriter)
+		if err != nil {
+			return m, err
+		}
+		return m.WithTermination(msg, nil), nil
+	}, padding: []int{1, 0, 0}})
```
Comment (RedbackThomson) [inline cmd/up/ctx/navigation.go:257]: nit: I'm not sure how I feel about the padding tbh. The option feels like it's floating awkwardly below.

**Review by RedbackThomson — APPROVED** [review]

File: cmd/up/ctx/navigation.go (hunk 3 above)
Comment (cwilhit) [reply cmd/up/ctx/navigation.go:257]: Agree, I'd prefer to have it inline with the rest

**Comment by RedbackThomson** [comment]
Will follow up with fixes in the next PR

**Pull request merged**